suse_migration_services/units/prepare.py

```diff
@@ -162,13 +172,14 @@ def main():
         system_mount.add_entry(
             zypp_plugins_services, '/usr/lib/zypp/plugins/services'
         )
-        if os.path.exists(cloud_register_metadata):
-            log.info('Bind mounting /var/lib/cloudregister')
-            Path.create('/var/lib/cloudregister')
+        if cloud_register_metadata and os.path.exists(cloud_register_metadata):
+            log.info('Bind mounting /var/cache/cloudregister from {0}'.format(
```

Comment:
maybe consider using a var for `/var/cache/cloudregister`
so you could do
```python
log.info("...{}...".format(cache_cloudregister_path),...)
Path.create(cache_cloudregister_path)
Command.run(
          [
                  ...
                   cache_cloudregister_path
          ]
)
```

Reply:
i will switch this out.